src/solver/expressions/include/antares/solver/expressions/visitors/NodeVisitor.h

```diff
@@ -19,18 +19,33 @@
 ** along with Antares_Simulator. If not, see <https://opensource.org/license/mpl-2-0/>.
 */
 #pragma once
+#include <functional>
+#include <iostream>
 #include <optional>
 #include <typeindex>
 #include <vector>
 
-#include <antares/logs/logs.h>
```

Comment:
About `#include <antares/logs/logs.h>` : 
As we know, this is the inclusion that causes a compilation error on Windows.
Just to be clear, this header indirectly `#include <windows.h>`, that holds an incompatibility with **antlr4**.
As a proof, replace `#include <antares/logs/logs.h>` with `#include <windows.h>` in the current **logs.h** causes the same compilation error.
Note that `#include <winsock2.h>` is also a cause for incompatibility, as this header includes **windows.h**